docs/development/flashing_debugging.md

```diff
@@ -30,7 +30,7 @@ Before you can flash the software, the device needs to be powered, either via US
 
 The same pins of the SWD connector on both boards should be connected with jumper wires. Pin 1 is usually marked with a small dot next to the connector. The following table gives an overview of the connector pinout:
 
-| Pin function     | Nucleo board   | Libre Solar board   |
+| Pin function     | Nucleo board (CN4)   | Libre Solar board   |
```

Comment:
Just checked all Nucleo boards I have and it seems that this is not *always* CN4. For the Nucleo with G0 series MCU on my desk it's CN11. So I'd rather keep it generic.

Or we could agree on "(usually CN4)".

Reply:
Let's just leave as it was before, this would add more confusion than help.